.github/workflows/ci.yml

```diff
@@ -52,10 +65,5 @@ jobs:
       - name: Install manim
         run: pip install . pytest
 
-      - name: Run tests (Non-Windows)
-        if: runner.os == 'macOS' || runner.os == 'Linux'
+      - name: Run tests
         run: pytest
-
-      - name: Run tests (Windows)
-        if: runner.os == 'Windows'
-        run: pytest --skip_end_to_end -rs
```

Comment:
I want just wondering, now that end to end tests are slow des with this PR why don’t we remove —skip_end_to_end flag ?

Reply:
https://github.com/Aathish04/manim/blob/windows-end-to-end/.github/workflows/ci.yml#L68-L69

I _did_ remove the `skip_end_to_end` flag. I think you may have seen the removed lines instead ;)

Reply:
Did you remove the @pytest.mark.skip_end_to_end flags everywhere ?

(Sorry, as I’m on mobile I can’t see)

Reply:
Oh _that's_  what you meant. I'm sorry, I misunderstood you. I'll remove them rn.

Reply:
Done :)